gatsby-node.js

```diff
@@ -93,7 +94,9 @@ async function fetchEndpoint(endpoint) {
       `http://${HOSTNAME}/open-disclosure/api/v1.0/${endpoint}`
     )
     if (response.ok) {
-      return await response.json()
+      return DUMMY_DATA[endpoint]
+
+      // return await response.json()
```

Comment:
I'd like to keep using real data for the other endpoints - mind switching this to only use fake data for referendums?

Reply:
Sure; I was getting errors about undefined not having fields.slug but that seemed to fix it. If you aren't having that issue I might have made a mistake setting up my back end somewhere then.

Reply:
Oh, it's probably because of the `elections[0]` below. If you're going to make that change in another PR then you can just re-enable `response.json()` there

Reply:
Ok nevermind, I'm not sure what's causing that error, that should be fixed with #135.